node/src/reactor/joiner.rs

```diff
@@ -229,6 +221,10 @@ pub enum Event {
     /// Chainspec loader announcement.
     #[from]
     ChainspecLoaderAnnouncement(#[serde(skip_serializing)] ChainspecLoaderAnnouncement),
+
+    /// Consensus request.
+    #[from]
+    ConsensusRequest(#[serde(skip_serializing)] ConsensusRequest),
```

Comment:
What type of consensus request do we make in the `joiner` phase?

Reply:
Ah, for status endpoint.

Reply:
The RpcServer can submit `ConsensusRequest::Status`. It's hard to make it possible to keep it in the joiner any other way.